database/server-delius-db.tf

```diff
@@ -35,7 +35,12 @@ module "delius_db" {
     database_global_database_name = "${var.ansible_vars_oracle_db["database_global_database_name"]}"
     database_sid                  = "${var.ansible_vars_oracle_db["database_sid"]}"
     database_characterset         = "${var.ansible_vars_oracle_db["database_characterset"]}"
-
+    database_type                 = "standalone"
```

Comment:
should this not be a variable?

Reply:
not in this case. it's a value which can't be derived from env config but is required for the DB module. This file is where the property is set.